src/framework/handlers/basis-worker.js

```diff
@@ -171,7 +167,7 @@ function BasisWorker() {
             // pvrtc
             case BASIS_FORMAT.cTFPVRTC1_4_RGB:
             case BASIS_FORMAT.cTFPVRTC1_4_RGBA:
-                return isPOT(width, height) && ((width === height) || webgl2);
+                return true;
```

Comment:
shouldn't this still be `return isPOT(width, height);` ?

Reply:
well spotted, thanks